sdk/azcore/core.go

```diff
@@ -14,7 +14,7 @@ import (
 // Request and react to the received Response.
 type Policy interface {
 	// Do applies the policy to the specified Request.  When implementing a Policy, mutate the
-	// request before calling req.Do() to move on to the next policy, and respond to the result
+	// request before calling req.Next() to move on to the next policy, and respond to the result
```

Comment:
Needs period at the end.

Reply:
There's one more line to the sentence which ends with a period.

Reply:
Ah yes...my eyes skipped straight over the next line.